product-service/handlers/createProduct.ts

```diff
@@ -0,0 +1,54 @@
+import { APIGatewayProxyHandler } from "aws-lambda";
+import { HTTPMethods, getHeaders } from "../utils/response-headers";
+import { Client } from "pg";
+import dbOptions from "../db/options";
+
+export const createProduct: APIGatewayProxyHandler = async (
+  event,
+  _context
+) => {
+  const { body, httpMethod } = event;
+
+  if (httpMethod !== HTTPMethods.POST) {
+    return { statusCode: 405, body: "Method Not Allowed" };
+  }
+
+  const client = new Client(dbOptions);
+  await client.connect();
+
+  try {
+    const { title, description, price, count } = JSON.parse(body);
+    const queryResult = await client.query(`
+      insert into products (title, description, price) values
+      ('${title}', '${description}', ${price}) returning *`);
+    const product = queryResult.rows[0];
+
+    await client.query(
+      `insert into stocks (product_id, count) values ('${product.id}', ${count})`
+    );
```

Comment:
Now try to post a product like this:
`{ "title": "','1',1); drop table products cascading; --" }`
...and have fun.

Reply:
My first SQL injection issue) I'm a frontend person, sorry) What is the best way to solve this problem?